app/views/helpers/index/normal/entry_bottom.phtml

```diff
@@ -42,7 +42,7 @@
 			<a class="dropdown-toggle" href="#dropdown-labels-<?php echo $this->entry->id();?>"><?php
 				echo _t('index.menu.tags');
 			?></a>
-			<ul class="dropdown-menu dropdown-menu-scrollable">
+			<ul class="dropdown-menu">
```

Comment:
I noticed that I've removed dropdown-menu-scrollable here. I'm not finding this in the master branch; is it something that got added in dev that shouldn't be removed?

Reply:
To find such things, I use the blame function:

https://github.com/FreshRSS/FreshRSS/blame/dev/app/views/helpers/index/normal/entry_bottom.phtml#L45

It points to https://github.com/FreshRSS/FreshRSS/pull/2108